README.md

```diff
@@ -91,3 +91,8 @@ Questions & Comments
 --------------------
 
 For any and all feedback, please use the Issues on this repository. 
+
+License
+--------------------
+
+By downloading the .zip you are agreeing to the terms in the project [EULA](http://aka.ms/dotnet-cli-eula).
```

Comment:
Shortened URL is pointing to the copy in ```master``` now. Let me know if it should be pointing to the ```rel/1.0.0``` copy.